code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -85,7 +85,9 @@
 	return
 
 /datum/antagonist/traitor/human/forge_traitor_objectives()
-	var/is_hijacker = prob(10)
+	var/is_hijacker = FALSE
+	if (GLOB.player_list.len > 30) // Less murderboning on lowpop thanks
```

Comment:
this needs to grab the amount of players actually playing the round

Reply:
@lzimann which glob is that
Also I euh considered that but there's usually plenty of late joins just after antags get assigned and people who AFK get disconnected after 15 mins anyways.

Reply:
`GLOB.joined_player_list`

Reply:
done